### PR TITLE
fix big keyframe markers in Dock_SoundLayer (and any other now)

### LINKED ISSUE #1601: Markers in Sound Panel are too big
**Synfig version & platform**:
1.3.15

**Issue description**:

screenshot_008

Details - https://forums.synfig.org/t/why-is-the-keyframe-marker-size-in-v-1-3-15-is-way-too-big-in-my-computer/11246/8

### PR DESCRIPTION
**Pull request by rodolforg**
Its default height was 64px. Now it's 10.
Besides, I also copied the Dock_Curves code for height updating.

fix #1601

### CODE REVIEW

**Comment by rodolforg** [comment]
@morevnaproject  now it's ready for urgent 1.3.16  :+1: 

I don't know how the hack I copied is useful yet.
On my Linux, it makes the Graph and Sound Panel a little different (few pixels larger) from Timetrack.

File: synfig-studio/src/gui/docks/dock_soundwave.cpp [hunk 1]
```diff
@@ -263,3 +266,26 @@ void Dock_SoundWave::setup_file_setting_data()
...
+	// COPIED FROM Dock_Curves
+	// FIXME very bad hack (timetrack dock also contains this)
+	//! Adapt the border size "according" to different windows manager rendering
+#ifdef _WIN32
+	header_height-=2;
+#elif defined(__APPLE__)
+	header_height+=6;
+#else
+// *nux and others
+	header_height+=2;
```
Comment (rodolforg) [inline synfig-studio/src/gui/docks/dock_soundwave.cpp:283]: This is unnecessary on my Linux

File: synfig-studio/src/gui/docks/dock_soundwave.cpp [hunk 2]
```diff
@@ -263,3 +266,26 @@ void Dock_SoundWave::setup_file_setting_data()
...
+#endif
+
+	int kf_list_height=10;
+
+	//widget_timeslider_.set_size_request(-1, header_height+1);
+	widget_timeslider.set_size_request(-1, header_height - kf_list_height + 5);
```
Comment (rodolforg) [inline synfig-studio/src/gui/docks/dock_soundwave.cpp:289]: As well as this + 5.

**Comment by morevnaproject** [comment]
> On my Linux, it makes the Graph and Sound Panel a little different (few pixels larger) from Timetrack.

I think this is not a problem. Thank you so much!

**Pull request merged**